### PR TITLE
Add RPM metadata only fixture

### PR DESCRIPTION
**Pull request by pavelpicka**
for ACS test

closes: #9422
https://pulp.plan.io/issues/9422

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -490,6 +493,10 @@ fixtures/rpm-signed: fixtures gnupghome
 fixtures/rpm-unsigned: fixtures
 	rpm/gen-fixtures.sh $@ rpm/assets
 
+fixtures/rpm-unsigned-meta-only: fixtures fixtures/rpm-unsigned
+	mkdir -p $@
+	cp -r fixtures/rpm-unsigned/repodata $@/
```
Comment (daviddavis) [inline Makefile:498]: I believe the `-r` option for cp is discouraged.

```suggestion
	cp -R fixtures/rpm-unsigned/repodata $@/
```

Comment (pavelpicka) [reply Makefile]: any mentioned in ```man``` but for consistency of ```Makefile``` looks much better.

Comment (daviddavis) [reply Makefile]: The behavior for `-r` is not defined by POSIX and can vary from system to system. On some systems, it's equivalent to `-R` but on other systems (e.g. FreeBSD) it's equivalent to `-RL`.

**Review by daviddavis — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (pavelpicka) [reply Makefile]: thank you

**Comment by ipanova** [comment]
looks good, can you add it to the fedora-all target as well, like https://github.com/pulp/pulp-fixtures/commit/f344457f949b0426524b29d0f5e8dc65a23c15b8

**Comment by ipanova** [comment]
disregard, i'm blind :)

**Review by ipanova — APPROVED** [review]

**Pull request merged**